.github/workflows/ci.yml

```diff
@@ -30,29 +26,42 @@ jobs:
     - name: Set display to virtual frame buffer
       run: export DISPLAY=:99.0
 
-
-
-    - name: Install npm dependencies
-      run: npm install
+    - name: Set up test environment
+      run: |
+        sudo apt-get update
+        sudo apt-get install xvfb gettext
+        wget https://github.com/mozilla/geckodriver/releases/download/v0.25.0/geckodriver-v0.25.0-linux64.tar.gz
+        mkdir geckodriver
+        tar -xzf geckodriver-v0.25.0-linux64.tar.gz -C geckodriver
+        export PATH=$PATH:$PWD/geckodriver
+        export BOKCHOY_HEADLESS=true
 
     - name: Install dependencies
       run: |
         pip install -r requirements/base.txt
         pip install -r requirements/testing.txt
 
-    - name: Run Python and Quality Tests
+    - name: Install Firefox 61.0
+      run: |
+        sudo apt-get purge firefox
+        wget "https://ftp.mozilla.org/pub/firefox/releases/46.0/linux-x86_64/en-US/firefox-46.0.tar.bz2"
```

Comment:
same here why do we need to install firefox to run `python` unit tests and `quality`